packages/app-form-builder/src/admin/plugins/formDetails/formSubmissions/FormSubmissionsList/useSubmissions.ts

```diff
@@ -96,29 +91,19 @@ export const useSubmissions = (form: Pick<FbFormModel, "id">) => {
         window.open(csvFile.src, "_blank");
     }, [form]);
 
-    const hasPreviousPage = state.page > 0;
-    const hasNextPage = typeof state.cursors[state.page + 1] === "string";
-
     return {
-        loading: state.loading,
-        refresh: () => loadSubmissions("network-only"),
-        submissions: state.submissions,
-        setSorter: (sort: string[]) => {
-            setState({ sort });
-        },
-        hasPreviousPage,
-        hasNextPage,
-        nextPage: () => {
-            if (!hasNextPage) {
+        loading: [listQuery].find(item => item.loading),
```

Comment:
Let's use `some` instead of `find`, that way the flag is boolean, and no `Boolean` needs to be used in [the other file](https://github.com/webiny/webiny-js/pull/3030/files#diff-0674c038a175333987e6672afe7cac92fbd6a669e7904234c81c0df5038f4fcfR92).